app/views/shared/_run_table.slim

```diff
@@ -24,7 +24,7 @@
           - if cols.include?(:uploaded)
             th.align-right = th_sorter('Uploaded', 'created_at')
       tbody
-        - runs.includes(:user).each do |run|
+        - runs.includes(:user, :game, :category).each do |run|
```

Comment:
When ctrl+f'ing the project, every single call to the partial had the name col, so it seemed fine to just include these always.